website/src/pages/team.tsx

```diff
@@ -120,6 +112,36 @@ const teamMembers = [
       "Previously Finance, Operations, and Product Strategy at Google & J.P. Morgan. Studied Commerce, Finance and Marketing at UVA.",
     image: "grace.jpg",
   },
+  {
+    name: "Fabian Meumertzheim",
+    email: "fabian@buildbuddy.io",
+    github: "fmeum",
+    twitter: "fhenneke",
+    linkedin: "fhenneke",
+    blurb:
+      "Previously Software Engineer at Code Intelligence. Maintainer of Bazel's rules_go and gazelle. Studied Mathematics and Computer Science at The University of Bonn.",
+    image: "fabian.jpg",
+  },
+];
+
+const emeriti = [
+  {
+    name: "Pari Parajuli",
+    email: "pari@buildbuddy.io",
+    github: "pariparajuli",
+    linkedin: "pari-parajuli",
+    blurb: "Software Engineering Intern studying Computer Science at University of California, Berkeley.",
+    image: "pari.png",
+  },
+  {
+    name: "Brentley Jones",
+    email: "brentley@buildbuddy.io",
```

Comment:
Do you want to remove these email addresses now?

Reply:
Good call, sent https://github.com/buildbuddy-io/buildbuddy/pull/6354